File: WebUI/Models/SvnDetails.cs
using System.Collections.Generic;
using SvnDomainModel;


public class SvnDetails
{
    public List<Svn.LogEntry> Changes { get; set; }
    public string TrunckPath {get; set;}
    public string BranchPath { get; set; }

    public SvnDetails()
    {
        Changes = new List<Svn.LogEntry>();
    }
}

File: SvnDomainModel/SvnDetails.vb

Public Class SvnDetails
    Public Property Changes As List(Of Svn.LogEntry)
    Public Property TrunckPath As String
    Public Property BranchPath As String
    Public Property SvnUser As String
    Public Property SvnPassword As String

    Public Const SvnCookieName = "SvnInfo"
    Public Const SvnCookieUserName = "SvnUser"
    Public Const SvnCookieUserPassword = "SvnPassword"

    Public Sub New()
        Changes = New List(Of Svn.LogEntry)
    End Sub
End Class
